server/views/pages/functionalSkills/index.njk

```diff
@@ -0,0 +1,75 @@
+{% extends "../../partials/layout.njk" %}
+
+{% set pageTitle = applicationName + " - Home" %}
+
+{% set pageId = "all-functional-skills" %}
+
+{% block beforeContent %}
+  {{ govukBreadcrumbs({
+    items: [
+      {
+        text: "Digital Prison Services",
+        href: dpsUrl
+      },
+      {
+        text: "Manage learning and work progress",
+        href: "/"
+      },
+      {
+        text: "Manage " + prisonerSummary.firstName | capitalize + " " + prisonerSummary.lastName | capitalize + "'s learning plan",
+        href: "/plan/" + prisonerSummary.prisonNumber + "/view/education-and-training"
+      }
+    ],
+    classes: 'govuk-!-display-none-print'
+  }) }}
+{% endblock %}
+
+{% block content %}
+
+  {% include "../../partials/prisonerBanner.njk" %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <span class="govuk-caption-l">Learning and work progress</span>
+      <h1 class="govuk-heading-l">{{ prisonerSummary.firstName | capitalize }} {{ prisonerSummary.lastName | capitalize }}'s functional skills</h1>
+
+      <table class="govuk-table" id="functional-skills-table">
+        <caption class="govuk-table__caption govuk-table__caption--s">Current functional skills</caption>
+
+        <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th scope="col" class="govuk-table__header">Subject</th>
+          <th scope="col" class="govuk-table__header">Assessed on</th>
+          <th scope="col" class="govuk-table__header">Level</th>
+          <th scope="col" class="govuk-table__header">Assessment type</th>
+          <th scope="col" class="govuk-table__header"><span class="govuk-visually-hidden">Action</span></th>
+        </tr>
+        </thead>
+
+        <tbody class="govuk-table__body">
+        {% for assessment in latestFunctionalSkills.assessments %}
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell">{{ assessment.type | formatFunctionalSkillType }}</td>
+            {% if assessment.assessmentDate and assessment.grade %}
+              <td class="govuk-table__cell">{{ assessment.assessmentDate | formatDate('D MMMM YYYY') }}</td>
+              <td class="govuk-table__cell">{{ assessment.grade }}</td>
+              <td class="govuk-table__cell">
+                Initial
+              </td>
+              <td class="govuk-table__cell">
+                <span class="govuk-hint">Curious</span>
```

Comment:
Because the word "Curious" is grey in the design, I've used the class "govuk-hint"
Is that right/appropriate?
I guess it is kinda a hint? 🤷